pubspec.yaml

```diff
@@ -278,6 +277,8 @@ dependency_overrides:
       url: https://github.com/linagora/flutter_file_picker
       ref: email_supported_5.3.1
 
+  json_annotation: 4.8.1
```

Comment:
Why do use it in `dependency_overrides`?

Reply:
otherwise we have to upgrade `json_annotation` in `jmap-dart-client` to `4.8.1`

Reply:
> otherwise we have to upgrade `json_annotation` in `jmap-dart-client` to `4.8.1`

So add it to `dependency_overrides` and don't remove it from `dependency`. It will cause warnings in other imports.

Reply:
Done